recipes/TXM_Sandbox/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "TXM_Sandbox" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/fa/57/4d28f168e88f19781c615a8b7a0566dffb1365550a3570c7289d8bbc0a07/TXM_Sandbox-0.1.tar.gz 
+  sha256: d9fd3cc1fa6b94235cfdb77aa66d3dc3f9d868fb2999a754ed62e47d01e2f2fb 
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - openjdk =8
+    - pyimagej
+    - nodejs
+    - xarray
+    - tomopy
+    - h5py
+    - tifffile
+    - imutils
+    - opencv
+    - openpyxl
+    - pytables
+    - pandas
+    - jupyter
+    - jupyterlab
+    - multiprocess
+    - scikit-image
+    - scikit-learn
+    - pystackreg
+    - silx
+    - napari 
```

Comment:
I can't see which run requirements are needed when looking at https://github.com/xianghuix/TXM_Sandbox/blob/master/setup.py but I guess you know.

Reply:
dependency libraries are added in setup.py file.

Reply:
This has no impact on the conda recipe, but note that you should only add the python packages (available on pypi) in your setup.py.
You shouldn't list `openjdk` in `setup.py` as it's not a python package.
conda on the other hand can install non python packages as well.